ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -254,12 +254,17 @@
       + "\nKSQL also marks its own internal topics as read-only. This is not controlled by this "
       + "config.";
 
+  public static final String KSQL_NEW_API_ENABLED = "ksql.new-api-enabled";
```

Comment:
This is a temporary property and will go away before KLIP-15 is complete.

Reply:
famous last words 😂  all joking aside, I've seen "temporary" things stay in the code base a long time. let's spend an extra minute now so that our public API (configs) are sane going forward